python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -715,14 +716,19 @@ def getMetadataCircle(coord, radius, filterName, epoch=None):
         md.add('RA', coord.getRa().asDegrees(), 'field center in degrees')
         md.add('DEC', coord.getDec().asDegrees(), 'field center in degrees')
         md.add('RADIUS', radius.asDegrees(), 'field radius in degrees, minimum')
-        md.add('SMATCHV', 1, 'SourceMatchVector version number')
-        md.add('FILTER', filterName, 'filter name for photometric data')
-        md.add('EPOCH', "NONE" if epoch is None else epoch.mjd, 'Epoch (TAI MJD) for catalog')
+        # Version 1: Initial version
+        # Version 2: JEPOCH for TAI Julian Epoch year of PM/parallax correction
+        md.add('SMATCHV', 2, 'SourceMatchVector version number')
+        md.add('FILTER', filterName, 'camera filter name for photometric data')
+        md.add('TIMESYS', "TAI", "time scale of time keywords")
+        md.add('JEPOCH', "NONE" if epoch is None else epoch.tai.jyear,
```

Comment:
FITS is going to really want JEPOCH to always be a floating point number. Can you please use `None` here instead? That would result in JEPOCH being an undefined value in the header which sounds like what you want.